changelog/issue-3823.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: patch
```

Comment:
I wonder if we should put `major` here, since anyone who runs public deployment would have to update the anonymous role to include this new scope. Otherwise it will break some functionality if role is not present in anonymous

@petemoore do you agree?